src/components/Layout/InputBasic.tsx

```diff
@@ -24,7 +24,7 @@ const InputBasic = ({ formValue, label, required = false, validation = {}, ...pr
   const errorMessage = errors[formValue]?.message?.toString() || ''
 
   return (
-    <FormControl isInvalid={!!errors[formValue]} isRequired={required}>
```

Comment:
I don't think you should have removed the isRequired here. Why did you remove it?